variables.tf

```diff
@@ -60,9 +60,9 @@ variable "rabbitmq_version" {
   validation {
     condition = anytrue([
       var.rabbitmq_version == null,
-      var.rabbitmq_version == "3.11"
+      var.rabbitmq_version == "3.9"
     ])
-    error_message = "Version must be 3.11. If no value passed, the current ICD preferred version is used."
+    error_message = "Version must be 3.9. If no value passed, the current ICD preferred version is used."
```

Comment:
why are you changing this to 3.9? 3.9 is no longer supported. Please revert this change. Only 3.11 is currently a supported version